File: test/attic/files/cli/test1/Main.scala
// @info no dependency
package test1
object Main {
  def main(args: Array[String]) = {
    val arg = if (args != null && args.length > 0) args(0) else "?"
    Console.println("1: test " + arg + " passed (" + args.length + ")")
  }
}

File: test/attic/files/cli/test1/Main.java
// @info no dependency
package test1;
public class Main {
  public static void main(String args[]) {
    String arg = (args.length > 0) ? args[0] : "?";
    System.out.println("1: test " + arg + " passed (" + args.length + ")");
  }
}
